docs/development.rst

```diff
@@ -515,16 +515,22 @@ patch updates are routinely and regularly released.
 
 This entails:
 
-1) Create a PR against ``master`` with the release notes;
-2) Once this has been merged, tag the release on GitHub with the
+1) Create a PR against ``master`` with the release notes; presently, the
+   release notes are located in ``docs/status.rst``
+2) Once this has been merged, tag the release on GitHub (on the `releases
+   <https://github.com/ga4gh/server/releases>`_ page) with the
    appropriate version number.
-3) Fetch the tag from the upstream repo, and checkout this tag.
-   Create the distribution tarball using ``python setup.py sdist``,
-   and then upload the resulting tarball to PyPI.
+3) Fetch the tag from the upstream repo, and checkout this tag.  Create the
+   distribution tarball using ``python setup.py sdist``, and then upload the
+   resulting tarball to PyPI using ``twine upload
+   dist/ga4gh-MAJOR.MINOR.PATCH.tar.gz`` (of course, using the correct file
+   name)
 4) Verify that the documentation at
-   http://ga4gh-reference-implementation.readthedocs.org/en/stable/
```

Comment:
Odd, I had to trigger the build for stable manually. It seems to be fine now. Have you got an account for RTD? I'll add you as a maintainer too so we don't have a single point of failure.

Anyway, this should still point to ``stable`` not latest. In RTD, latest==master, and stable==last tagged version